assets/scss/_custom.scss

```
@@ -26,6 +26,20 @@ $announcement-size-adjustment: 8px;
   }
 }
 
+/* Fix misplaced scrolling to targets with narrow
+   viewports
+*/
+@media only screen and (max-width: 768px) {
+  body {
+    scroll-padding: 16rem 0 0 0;
+  }
+
+  .td-main {
+    padding-top: 4rem;
+  }
+}
+
+
```

```suggestion
html {
  @media only screen and (max-width: 768px) {
    scroll-padding-top: 5rem;
  }
  @media only screen and (max-width: 1024px) {
    scroll-padding-top: 4rem;
  }
}
```

Somehow adding the padding to the body doesn't have any effect. With HTML, it works. This solution seems to works.